server/src/main/java/io/seata/server/coordinator/DefaultCoordinator.java

```diff
@@ -104,6 +108,11 @@
      */
     protected int timeoutRetryDelay = CONFIG.getInt(ConfigurationKeys.TIMEOUT_RETRY_DELAY, 5);
 
+    /**
+     * The Timeout retry delay.
+     */
+    protected int transactionUndologDeleteDelay = CONFIG.getInt(ConfigurationKeys.TRANSACTION_UNDO_LOG_DELETE_DELAY, 24 * 60);
```

Comment:
best to indicate the time unit

Reply:
same with committingRetryDelay,asynCommittingRetryDelay,timeoutRetryDelay, default is SECONDS